kotlinx-coroutines-core/common/src/Builders.common.kt

```diff
@@ -153,6 +153,7 @@ public suspend fun <T> withContext(
         // FAST PATH #1 -- new context is the same as the old one
         if (newContext === oldContext) {
             val coroutine = ScopeCoroutine(newContext, uCont)
+            @Suppress("LEAKED_IN_PLACE_LAMBDA") // Contract is preserved, invoked immediately or throws
```

Comment:
This one (an one on the contract definition) is annoying, I'll report it to the team